cmd/node/config/testKeys/validatorKey.pem

```diff
@@ -0,0 +1,4 @@
+-----BEGIN PRIVATE KEY for 75907bf66c64949f8d1dd003ed8fd815e8dec189a9e50e4bfd7e2592cfdc92641baf500e3533820e0f58c4ed3f39c01750f0017726731fe5bc66a0ab6fc9a7b5661d163e72f099fe29df16fd59ded198dc1423575be1b6e2160e0d84a5ece08a-----
```

Comment:
I think we need another approach here.
We can leave it as it is and I will fix this in an upcoming PR